molecule_docker/playbooks/create.yml

```diff
@@ -40,14 +40,22 @@
           {%- else -%}
           {{ playbook_dir + '/Dockerfile.j2' }}
           {%- endif -%}
-        dest: "{{ molecule_ephemeral_directory }}/Dockerfile_{{ item.image | regex_replace('[^a-zA-Z0-9_]', '_') }}"
+        dest: "{{ item.path | default(molecule_ephemeral_directory) }}/Dockerfile_{{ item.image | regex_replace('[^a-zA-Z0-9_]', '_') }}"
         mode: "0600"
       loop: "{{ molecule_yml.platforms }}"
       loop_control:
         index_var: i
       when: not item.pre_build_image | default(false)
       register: platforms
 
+    - name: Copy files for image building
+      copy:
```

Comment:
I am curious if use of syncronize would not be better for this use case, at it would avoid copying the files each time. @tadeboro WDYT?

Reply:
I did not use the synchronize module because it requires the rsync to be installed, and copying files locally should not be too much of a bottleneck. If people start hitting performance issues here, they should really prepare the images for molecule in advance.

But in the end, I do not mind changing this to use rsync since I do not use that functionality at all.